trex-container-app/server/Dockerfile

```
@@ -22,4 +29,9 @@ ENV PYTHONPATH="/opt/trex/trex-core/scripts/automation/trex_control_plane/intera
 WORKDIR /opt/trex/trex-core/scripts
 ENV TREX_DIR="/opt/trex/trex-core/scripts"
 ENV TRAFFICGEN_DIR="/opt/trafficgen"
-COPY scripts /usr/local/bin/
+
+# Copy scripts
+COPY scripts /usr/local/bin
+COPY --from=build /scripts/webserver /usr/local/bin/
+
+RUN chmod +x /usr/local/bin/webserver
```

Avoid copying anything that is not needed in the image that is inside the scripts, e.g. the webserver.go

```suggestion
COPY --chmod=550 --from=build /scripts/webserver /usr/local/bin/
```

---

done